client/src/components/ChatInput.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { TextField } from "@material-ui/core";
+import InsertEmoticonOutlinedIcon from "@material-ui/icons/InsertEmoticonOutlined";
+import PhotoLibraryOutlinedIcon from "@material-ui/icons/PhotoLibraryOutlined";
```

Comment:
Feel free to rename these to something smaller, like 
`import LibraryIcon from "@material-ui/icons/PhotoLibraryOutlined";`